langchain/src/chat_models/tests/chatllama_cpp.int.test.ts

```diff
@@ -85,14 +85,57 @@ test.skip("Test chain with memory", async () => {
 test.skip("test streaming call", async () => {
   const llamaCpp = new ChatLlamaCpp({ modelPath: llamaPath, temperature: 0.7 });
 
+  const stream = await llamaCpp.stream(
+    "Tell me a short story about a happy Llama."
+  );
+
+  const chunks = [];
+  for await (const chunk of stream) {
+    chunks.push(chunk.content);
+    console.log(chunk.content);
+  }
+
+  expect(chunks.length).toBeGreaterThan(1);
+});
+
+test.skip("test multi-mesage streaming call", async () => {
```

Comment:
fyi if you were unaware, we don't run `int` tests in CI so no need to add `skip`